app/assets/javascripts/member-facing/backbone/fundraiser.js

```diff
@@ -46,21 +49,32 @@ const Fundraiser = Backbone.View.extend(_.extend(
     this.changeStep(1);
     this.donationAmount = 0;
     this.followUpUrl = options.followUpUrl;
-    this.submissionCallback = options.submissionCallback;
+    if (typeof options.submissionCallback === 'function') {
+      this.submissionCallback = options.submissionCallback;
+    }
+
     this.initializeSkipping(options);
     this.pageId = options.pageId;
     this.directDebitOpened = false;
     this.displayDirectDebit(options.showDirectDebit);
-    this.initializeRecurring(options.recurringDefault);
     this.updateButton();
+
+    this.paymentMethods = new Backbone.Collection(options.paymentMethods || []);
+    this.paymentMethodsView = new PaymentMethodsView({ collection: this.paymentMethods });
+
+    this.setOneClickVisibility();
+    this.initializeRecurring(options.recurringDefault);
+
     GlobalEvents.bindEvents(this);
+
   },
 
   initializeRecurring(recurringDefault) {
-    const $checkbox = this.$('input.fundraiser-bar__recurring');
+    const $checkbox = this.$('input.fundraiser-bar__recurring, input.fundraiser-bar__recurring-one-click');
+
     switch(recurringDefault) {
       case 'only_recurring':
-        $checkbox.parents('.form__group').addClass('hidden-irrelevant');
+        $checkbox.parents('label').addClass('hidden-irrelevant');
```

Comment:
If you put each one it's own form group, the spacing should be consistent between form elements